setContentDescriptor.sh

```diff
@@ -6,4 +6,4 @@ git diff  --diff-filter=D $2 > delete-changelog.txt
 ASSETID=$1
 VERSION=$3
 
-python release_notes.py  $VERSION changelog.txt delete-changelog.txt $ASSETID
\ No newline at end of file
+python Tests/release_notes.py  $VERSION changelog.txt delete-changelog.txt $ASSETID
```

Comment:
It is not a test
It is a tool that is part of our build process, it  creates our release notes